app/Policies/PostPolicy.php

```diff
@@ -22,7 +21,7 @@ public function __construct()
     /**
      * Determine if the full post should be displayed.
      *
-     * @param  Illuminate\Contracts\Auth\Authenticatable $user
+     * @param  $user
```

Comment:
Let's leave the docblock since it's handy to know what this will be (even though PHP won't enforce it). When we upgrade Rogue to use PHP 7.1 at some point, this would be a great use-case for [nullable types](http://php.net/manual/en/migration71.new-features.php#migration71.new-features.nullable-types), e.g:

```php
public function viewAll(?Authenticatable $user, Post $post) { // ...
```